README.md

```diff
@@ -31,12 +31,13 @@ List of dependencies:
  - yojson >= 1.6.0
  - cmdliner
  - ppx\_inline\_test
+ - why3
```

Comment:
Add version requirement.

Reply:
Added in b8fb9af3a00c2fbb135c1dbd2e38fb315051079c